static/app/utils/performance/contexts/pageError.tsx

```diff
@@ -1,17 +1,23 @@
 import {createContext, useContext, useState} from 'react';
+import {Theme} from '@emotion/react';
 
 import {Alert} from 'sentry/components/alert';
 
+export type PageAlert = {
+  message: string;
+  type: keyof Theme['alert'];
```

Comment:
feels weird that this is still called `pageError`.

Should we refactor to call this `pageAlert` now?

Reply:
Yeah I 100% agree, I'll refactor in another PR!